personalAccessKey.js

```diff
@@ -69,15 +71,15 @@ async function refreshAccessToken(
   });
   writeConfig();
 
-  return accessToken;
+  return accessTokenResponse;
 }
 
-async function getNewAccessToken(accountId, personalAccessKey, expiresAt, env) {
+async function getNewAccessToken(accountId, personalAccessKey, expiresAt, env, fullAPIResponse=false) {
```

Comment:
Previous implementation to get the `enabledFeatures` fetched to refresh the access key but didn't write it to the config if it got a new one - fixes that